rickchurch/auth.py

```diff
@@ -126,7 +126,7 @@ async def add_user(user: dict, db_conn: asyncpg.Connection) -> str:
     user_id = int(user["id"])  # User id (snowflake) from discord
 
     async with db_conn.transaction():
-        user_data = await db_conn.fetchrow("SELECT * FROM users WHERE user_id = $1")
+        user_data = await db_conn.fetchrow("SELECT * FROM users WHERE user_id = $1", user_id)
```

Comment:
Would this prevent SQL injection?

Reply:
Yes, passing arguments with this function is safe, using f-strings or other forms of string concatenation wouldn't be, but this is indeed safe.

Reply:
👌🏾,  I was just checking! Good job!